NEWS

```diff
@@ -114,6 +114,10 @@ CHANGES WITH 256:
         * systemd.crash_reboot and related settings are deprecated in favor of
           systemd.crash_action=.
 
+        * Stable releases for version v256 and newer will now be pushed in the
+          main repository. The systemd-stable repository will be used for existing
+          stable branches, and when they reach EOL it will be archived.
```

Comment:
Maybe explicitly say what those "existing" stable branches are, to make it easier for people to understand without context. "for existing stable branches (`v255-stable` and lower)"?

Also "pushed in the main repo" sounds a bit strange… One usually pushes drugs in a shady neighbourhood. Maybe "available in" or "part of"?